src/components/LanguageEditor/LanguageEditor.test.js

```diff
@@ -0,0 +1,24 @@
+describe(`LanguageEditor`, () => {
+
+  beforeEach( () => {
+    cy.visit(`/#/langauges`);
```

Comment:
I do this _all_ the time. I regularly search projects for this typo out of habit now 😄.

```suggestion
    cy.visit(`/#/languages`);
```